app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/mbentity/Label.java

```diff
@@ -4,16 +4,37 @@
 
 import org.metabrainz.mobile.data.sources.api.entities.LifeSpan;
 
+import java.util.ArrayList;
+
 public class Label {
     @SerializedName("id")
     private String mbid;
     private String name;
     private String disambiguation;
     private String type;
+    @SerializedName("label-code")
+    private String code;
     @SerializedName("life-span")
     private LifeSpan lifeSpan;
     private String country;
     private Area area;
+    private ArrayList<Release> releases = new ArrayList<>();
+
+    public ArrayList<Release> getReleases() {
+        return releases;
+    }
+
+    public void setReleases(ArrayList<Release> releases) {
+        this.releases = releases;
+    }
+
+    public String getCode() {
+        return "LC " + code;
```

Comment:
To prevent showing `LC null` when there's no code:

![lc-null](https://user-images.githubusercontent.com/465723/56959102-9e5b9780-6b4c-11e9-95e3-2ad5b097fac2.png)


```suggestion
        if (code != null) return "LC " + code;
        return "";
```